lib/ship.rb

```diff
@@ -0,0 +1,19 @@
+class Ship
+  attr_reader :name,
+              :length,
+              :health
+
+  def initialize(name, length)
+    @name = name
+    @length = length
+    @health = length
```

Comment:
We changed health method to integrate into initialize method and added health to attr_reader